build.gradle

```diff
@@ -1,5 +1,6 @@
 buildscript {
     repositories {
+        jcenter()
```

Comment:
:+1: for jcenter but I read about people having problems with missing libs in jcentral which ones were present in maven central, so I think we can have both of them.